### PR TITLE
Working Groups clean-up

### PR DESCRIPTION
**Pull request by jdorfman**

### CODE REVIEW

File: _data/wgs.yml [hunk 1]
```diff
@@ -72,77 +74,80 @@
...
   description: "What are the core set of principles for what authentic participation means in multiple contexts (e.g. corporate, individual, humanitarian/NGOs, etc.), as it relates to OSS code?"
 
 - title: Transparency Goals
   bottomliner: Duane O'Brien
   slug: transparency-goals
-  status: Active
-  ranking: 2
+  status: Incubating
+  ranking: 0
   description: "Good corp, bad corp: what does sustainability and ethical practices mean for corporations who run OSPOs or who use open source code?"
```
Comment (jwflory) [inline _data/wgs.yml]: Not sure if @DuaneOBrien agrees, but the Principles of Authentic Participation working group came from Duane's Transparency Goals sessions at Sustain. Maybe this is in the direction of accountability though, which we haven't explored in depth on P.A.P. working group.

File: _data/wgs.yml [hunk 2]
```diff
@@ -2,68 +2,70 @@
   bottomliner: Farbod Saraf
   slug: leadership
   status: Incubating
-  ranking: 1
+  ranking: 0
```
Comment (RichardLitt) [inline _data/wgs.yml]: This hasn't had it's first meeting yet. Keep it in incubating.

File: _data/wgs.yml [hunk 3]
```diff
@@ -2,68 +2,70 @@
...
   description: "What does it take to be an open source leader? How do leaders emerge, and what can we learn about what makes leaders effective? What are the pressures and issues that leaders face, and how can we mitigate them through active development?"
 
 - title: Open Source Funders
   bottomliner: Richard Littauer
   slug: open-source-funders
   status: Incubating
-  ranking: 1
+  ranking: 0
```
Comment (RichardLitt) [inline _data/wgs.yml]: Keep it in incubating. This is not active.

File: _data/wgs.yml [hunk 4]
```diff
@@ -2,68 +2,70 @@
...
   description: "There's no current database of how much money goes into open source: we want to build this. Follow the money - where does it come from, where does it go, and how useful are different avenues of funding for open source projects at scale."
 
 - title: Sustainability Readiness
   bottomliner: Erin McKean
   slug: sustainability-readiness
   status: Incubating
-  ranking: 1
+  ranking: 0
```
Comment (RichardLitt) [inline _data/wgs.yml]: This hasn't had it's first meeting yet. Keep it in incubating.

File: _data/wgs.yml [hunk 5]
```diff
@@ -2,68 +2,70 @@
...
   description: "How can we ensure that a new open source project is born sustainable? What steps can be put into place to fight entropy in both code and governance structures in open source?"
 
 - title: Licensing
   bottomliner: Benjamin Nickolls
   slug: licensing
   status: Incubating
-  ranking: 1
+  ranking: 0
```
Comment (RichardLitt) [inline _data/wgs.yml]: This hasn't had it's first meeting yet. Keep it in incubating.

File: _data/wgs.yml [hunk 6]
```diff
@@ -2,68 +2,70 @@
...
   description: "What is licensing, anyway? What's the current state of the ecosystem, where are we going, and how can we talk about licenses without flipping over the proverbial dinner table?"
 
 - title: Personas on Sustainability
   bottomliner: Jacob Redding
   slug: personas
   status: Incubating
-  ranking: 1
+  ranking: 0
```
Comment (RichardLitt) [inline _data/wgs.yml]: This hasn't had it's first meeting yet. Keep it in incubating.

File: _data/wgs.yml [hunk 7]
```diff
@@ -2,68 +2,70 @@
...
   description: "Who are sustainers, anyway? What does it mean to sustain a project? How can we use persona development to learn more about the needs of all of the stakeholders in open source?"
 
 - title: 2020 Sustain Summit Report
   bottomliner: Benjamin Nickolls
   slug: 2020-summit-report
   status: Incubating
-  ranking: 1
+  ranking: 0
```
Comment (RichardLitt) [inline _data/wgs.yml]: This hasn't had it's first meeting yet, and no work has been done. Keep it in incubating.

File: _data/wgs.yml [hunk 8]
```diff
@@ -2,68 +2,70 @@
...
   description: "It was great, everyone loved it, but - what happened at the 2020 Sustai Summit for those of us who weren't there, or who were busy thinking about a couple of hard problems in open source? How can we further Sustain's goals, going forward?"
 
 - title: Academic and Specialized Projects
   bottomliner: Yo Yehudi
   slug: academic-projects
   status: Incubating
-  ranking: 1
+  ranking: 0
```
Comment (RichardLitt) [inline _data/wgs.yml]: This hasn't had it's first meeting yet, and isn't close to getting off of the ground. Keep it in incubating.

File: _data/wgs.yml [hunk 9]
```diff
@@ -2,68 +2,70 @@
...
   description: "Open source doesn't just mean web development. How does sustainability relate to academic and specialized projects that have different needs and users?"
 
 - title: Design and UX
   bottomliner: Georgia Bullen
   slug: design-and-ux
   status: Incubating
-  ranking: 1
+  ranking: 0
```
Comment (RichardLitt) [inline _data/wgs.yml]: This hasn't had it's first meeting yet. Keep it in incubating.

File: _data/wgs.yml [hunk 10]
```diff
@@ -2,68 +2,70 @@
...
   description: "How can we make open source design and user experience sustainable, too? What does sustainability mean for code-adjacent territories?"
 
 - title: Docs
   bottomliner: Chris Chinchilla, Pia Mancini
   slug: docs
+  url: true
```
Comment (RichardLitt) [inline _data/wgs.yml]: https://sustainoss.org/working-groups/docs/ is not ready. There's many empty fields. url should remain false.

File: _data/wgs.yml [hunk 11]
```diff
@@ -2,68 +2,70 @@
...
   status: Incubating
   ranking: 1
   description: "Who writes, uses, and updates the docs? What do sustainable-first docs look like? How can we write better ones?"
 
 - title: Ethics
   bottomliner: Chris Chinchilla
   slug: ethics
+  url: true
```
Comment (RichardLitt) [inline _data/wgs.yml]: https://sustainoss.org/working-groups/ethics is not ready yet, and should not have been merged. Keep URL as false.

File: _data/wgs.yml [hunk 12]
```diff
@@ -72,77 +74,79 @@
   bottomliner: Agost Biro
   slug: business-models
   status: Incubating
-  ranking: 1
+  ranking: 0
```
Comment (RichardLitt) [inline _data/wgs.yml]: This hasn't had it's first meeting yet. Keep it in incubating.

File: _data/wgs.yml [hunk 13]
```diff
@@ -72,77 +74,79 @@
...
   description: "For individual developers, what does sustainability look like? How can developers get money for their projects, taking them out of the abandon-ware and hobbyist stage?"
 
 - title: Membership models for OSS
   bottomliner: Gabriela Rodríguez Berón
   slug: membership-models
   status: Incubating
-  ranking: 1
```
Comment (RichardLitt) [inline _data/wgs.yml]: This hasn't had it's first meeting yet. Keep it in incubating.

File: _data/wgs.yml [hunk 14]
```diff
@@ -72,77 +74,79 @@
...
+  ranking: 0
   description: "What do models of membership look like, specifically for organisations developing FLOSS? How can we all be involved?"
 
 - title: Climate Emergency
   bottomliner: Xavier Damman
   slug: climate-emergency
   status: Incubating
-  ranking: 1
```
Comment (RichardLitt) [inline _data/wgs.yml]: This hasn't had it's first meeting yet. Keep it in incubating.

File: _data/wgs.yml [hunk 15]
```diff
@@ -72,77 +74,79 @@
...
+  ranking: 0
   description: "Sustainability doesn't just mean code. How can we make our work sustainable environmentally, too? How can we share our knowledge of open source with other groups in the world?"
 
 - title: Community Principles & Ostrom Redux
   bottomliner: Greg Bloom
   slug: ostrom
-  status: Active
-  ranking: 2
+  status: Incubating
```
Comment (RichardLitt) [inline _data/wgs.yml]: This hasn't had it's first meeting yet. Keep it in incubating.

File: _data/wgs.yml [hunk 16]
```diff
@@ -72,77 +74,79 @@
...
+  ranking: 0
   description: "What does an open source developer have to do to find new users, more sponsors, and better swag? What is sustainable marketing for open source?"
 
 - title: Principles of Authentic Participation
   bottomliner: Justin W. Flory
   slug: authentic-participation
   url: true
   status: Active
-  ranking: 2
+  ranking: 1
```
Comment (RichardLitt) [inline _data/wgs.yml]: This should be ranked as 0.

File: _data/wgs.yml (hunk 1 above)
Comment (RichardLitt) [reply _data/wgs.yml]: This is a different group. I'm waiting for Duane's PR. I wouldn't touch it, for now.

File: _data/wgs.yml [hunk 17]
```diff
@@ -72,77 +74,79 @@
...
+  status: Incubating
+  ranking: 0
   description: "Good corp, bad corp: what does sustainability and ethical practices mean for corporations who run OSPOs or who use open source code?"
 
 - title: Motivating Contributors
   bottomliner: Richard Littauer
   slug: motivating-contributors
   status: Incubating
-  ranking: 1
+  ranking: 0
```
Comment (RichardLitt) [inline _data/wgs.yml]: We've had one meeting, but there's still work to do to make this not just incubating.

File: _data/wgs.yml [hunk 18]
```diff
@@ -72,77 +74,79 @@
...
   description: "How can we give back to OSS projects that we already use at our work? What does our stack look like, and who among our dependencies is looking for funding?"
 
 - title: FOSS Responders
   bottomliner: Duane O'Brien, Megan Byrd-Sanicki, and Richard Littauer
   slug: foss-responders
-  status: Active
-  ranking: 2
+  status: Incubating
+  url: true
+  ranking: 1
```
Comment (RichardLitt) [inline _data/wgs.yml]: No, this is active. What do you think incubating means?

File: _data/wgs.yml (hunk 2 above)
Comment (jdorfman) [reply _data/wgs.yml]: @RichardLitt ah, my bad, we should document that.

Comment (RichardLitt) [reply _data/wgs.yml]: Hmm. Document what? That it hasn't had a meeting yet? It currently says that on the website. Do you want me to document it, somewhere else?

Comment (jdorfman) [reply _data/wgs.yml]: nah all good. My bad!

**Pull request closed without merging**